corehq/apps/app_manager/tests/test_filters.py

```diff
@@ -0,0 +1,133 @@
+import datetime
+from collections import namedtuple
+from contextlib import contextmanager
+from django.test import SimpleTestCase
+from mock import patch
+from corehq.apps.app_manager.models import CustomMonthFilter
+
+
+Date = namedtuple('Date', ('year', 'month', 'day'))
+
+
+MAY_15 = Date(2015, 05, 15)
+MAY_20 = Date(2015, 05, 20)
+MAY_21 = Date(2015, 05, 21)
```

Comment:
er, maybe it still works, though I think it's still weird/misleading since these will be interpreted as [octals](http://stackoverflow.com/questions/11620151/what-do-numbers-starting-with-0-mean-in-python)